syne_tune/optimizer/schedulers/searchers/bayesopt/models/estimator.py

```diff
@@ -174,31 +174,3 @@ def transform_state_to_data(
         targets = np.vstack([targets * np.ones((1, num_fantasy_samples))] + fanta_lst)
         features = np.vstack([features] + cand_lst)
     return TransformedData(features, targets, mean, std)
-
-
-class EstimatorFromTransformedData(Estimator):
```

Comment:
Why remove this one? I thought this is something we wanted to have

Reply:
EstimatorWrapper now implements this function, its a rename + move rather than a removal.